src/plugins/explore/public/components/visualizations/utils/use_debounced_value.ts

```diff
@@ -94,5 +94,5 @@ export function useDebouncedNumericValue(
     setDebouncedValue(numValue);
   };
 
-  return [localValue, handleNumericChange];
+  return [localValue.toString(), handleNumericChange];
```

Comment:
Is this expected? The function name is `useDebouncedNumericValue`

Reply:
I see, thanks for sharing, I have a PR to refactor these Debounced* components, will take this into consideration, I think you can leave it as it is for now